homeassistant/components/recorder/core.py

```diff
@@ -1417,6 +1419,8 @@ def _end_session(self) -> None:
         if self.event_session is None:
```

Comment:
We should make `self.event_session` protected -> `self._event_session` in the future so we don't have any mistakes